web/src/reducers/activities.selectors.js

```diff
@@ -2,15 +2,14 @@ import { createSelector } from 'reselect';
 import { selectApdData } from './apd.selectors';
 import { stateDateRangeToDisplay, stateDateToDisplay } from '../util';
 
-export const selectActivityKeys = ({ activities: { allKeys } }) => allKeys;
-
-export const selectActivitiesByKey = ({ activities: { byKey } }) => byKey;
-
 export const selectActivityByKey = ({ activities: { byKey } }, { aKey }) =>
   byKey[aKey];
 
-export const selectAllActivities = ({ activities: { byKey } }) =>
-  Object.values(byKey);
+export const selectAllActivities = ({
```

Comment:
For this selector, get activities from the APD instead of that weird `byKeys` thing we add.